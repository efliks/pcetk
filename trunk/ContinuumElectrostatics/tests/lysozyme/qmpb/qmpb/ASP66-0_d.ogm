ON_GEOM_CENT 121 2.0
(10.020 11.668 27.100)  101  1.000
(10.020 11.668 27.100)  101  0.500
(10.020 11.668 27.100)  101  0.250
