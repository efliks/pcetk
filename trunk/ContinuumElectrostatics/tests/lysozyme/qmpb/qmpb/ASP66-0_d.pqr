../0/site_ASP_66_d.pqr