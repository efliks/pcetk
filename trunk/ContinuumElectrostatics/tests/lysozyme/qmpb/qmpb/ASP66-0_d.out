Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named ASP66-0_d
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.919373, 20.3963, 19.0418)) 121 2
Finer level:    (10.02, 11.668, 27.1) 101 1
Finer level:    (10.02, 11.668, 27.1) 101 0.5
Finest level:   (10.02, 11.668, 27.1) 101 0.25


Self energy of ASP66-0_d in three dielectric environment = 322.675
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of ASP66-0_d with background charge set in three dielectric environment = -15.4304
(probably in kcal/mole)



Interaction energies of ASP66-0_d with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.0150229
0   1 : -0.000125924
1   0 : 0.000360466
1   1 : 0.0118351
2   0 : -0.00318506
2   1 : -6.35563e-05
3   0 : -0.0223869
3   1 : 0.000316735
3   2 : 0.000343519
3   3 : 0.0225517
4   0 : -0.00123636
4   1 : 0.00437201
5   0 : -0.00175064
5   1 : 0.019625
6   0 : 0.0016057
6   1 : 0.0142387
7   0 : -0.0142142
7   1 : -0.000274167
8   0 : -0.0087947
8   1 : 0.106749
9   0 : 0.0298265
9   1 : 0.395267
10   0 : -0.0369346
10   1 : 0.592429
11   0 : -3.43954
11   1 : 5.4922
12   0 : 393.298
12   1 : 645.35
13   0 : -0.00330901
13   1 : 0.0284333
14   0 : -0.0214005
14   1 : -0.000451482
15   0 : -0.0405525
15   1 : 0.00020118
16   0 : 0.00116909
16   1 : 0.0594176
17   0 : -0.0104544
17   1 : 9.54423e-05
18   0 : 6.05203e-05
18   1 : 0.00266709
Total runtime: 7.61 s user 0.05 s system 
